### PR TITLE
Edge to edge in starter branch

### PR DESCRIPTION
**Pull request by android-dev-lxl**

### CODE REVIEW

File: app/src/main/java/com/example/inventory/ui/item/ItemEntryScreen.kt [hunk 1]
```diff
@@ -33,6 +35,7 @@ import androidx.compose.material3.Text
 import androidx.compose.material3.TextFieldDefaults
```
Comment (johnshea) [inline app/src/main/java/com/example/inventory/ui/item/ItemEntryScreen.kt:35]: remove unused import

File: app/src/main/java/com/example/inventory/ui/item/ItemEntryScreen.kt
Comment (johnshea) [inline app/src/main/java/com/example/inventory/ui/item/ItemEntryScreen.kt:1]: ln 187 - arguments should be on their own lines

File: app/src/main/java/com/example/inventory/ui/home/HomeScreen.kt
Comment (johnshea) [inline app/src/main/java/com/example/inventory/ui/home/HomeScreen.kt:1]: ln 201 - fix identing

**Review by johnshea — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/inventory/ui/theme/Theme.kt [hunk 2]
```diff
@@ -112,16 +111,15 @@ fun InventoryTheme(
         darkTheme -> DarkColorScheme
         else -> LightColorScheme
     }
+
     val view = LocalView.current
```
[the remark this answers is not in the document]
Comment (android-dev-lxl) [reply app/src/main/java/com/example/inventory/ui/theme/Theme.kt:115]: Added to main branch

**Review by johnshea — APPROVED** [review]

**Pull request merged**